src/libstd/net/ip.rs

```diff
@@ -778,21 +787,48 @@ impl From<Ipv4Addr> for u32 {
 
 #[stable(feature = "ip_u32", since = "1.1.0")]
 impl From<u32> for Ipv4Addr {
-    /// It performs the conversion in network order (big-endian).
+    /// Convert an host byte order `u32` into an `Ipv4Addr`.
```

Comment:
nit: I think this should be "***a*** host byte [...]" (At least it should be consistent with the doc comment above).

Reply:
good catch! addressed in the latest force push